devtools/x/src/lint/guppy.rs

```diff
@@ -507,3 +507,167 @@ impl PackageLinter for OnlyPublishToCratesIo {
         Ok(RunStatus::Executed)
     }
 }
+
+// Ensure that Move crates do not depend on Diem crates.
+#[derive(Debug)]
+pub struct MoveCratesDontDependOnDiemCrates;
+
+// List of existing move to diem dependencies.
+// The goal is eliminate all of them gradually.
+// In general, you should refrain from adding new entries to this list.
+const EXISTING_MOVE_TO_DIEM_DEPS: &[(&str, &str)] = &[
```

Comment:
Might be better to stick this in a field in x.toml?

Reply:
Didn't know you could do that! Let me try

Reply:
Yeah it would be great to have this in x.toml :)

Reply:
Done.